<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_0">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="interfaceGroupBox">
         <property name="title">
          <string>Interface:</string>
         </property>
         <layout class="QFormLayout" name="formLayout_8">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Language:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cboBoxLang"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="delayGroupBox">
         <property name="title">
          <string>Delay:</string>
         </property>
         <layout class="QFormLayout" name="formLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="mainDelayLabel">
            <property name="text">
             <string>Main screen delay (ms):</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="PhLockableSpinBox" name="mainScreenDelayspinBox">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Second screen delay (ms):</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="PhLockableSpinBox" name="secondScreenDelaySpinBox">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="pipGroupBox">
         <property name="title">
          <string>Picture in picture:</string>
         </property>
         <layout class="QFormLayout" name="formLayout_6">
          <item row="1" column="0">
           <widget class="QLabel" name="pipRatioLabel">
            <property name="text">
             <string>Ratio:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="pipOffsetLabel">
            <property name="text">
             <string>Offset (ms):</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="pipOffsetSpinBox">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="pipRatioSlider">
            <property name="maximum">
             <number>50</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="sliderPosition">
             <number>30</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QRadioButton" name="pipLeftPositionRadioButton">
            <property name="text">
             <string>Left</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="pipPositionLabel">
            <property name="text">
             <string>Position:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QRadioButton" name="pipRightPositionRadioButton">
            <property name="text">
             <string>Right</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabStrip">
      <attribute name="title">
       <string>Strip</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lblSpeed">
         <property name="text">
          <string>Speed:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinBoxSpeed">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="fontLabel">
         <property name="text">
          <string>Font:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QFontComboBox" name="fontComboBox"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblStripHeight">
         <property name="text">
          <string>Strip Height</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSlider" name="sliderStripHeight">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>22</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSynchro">
      <attribute name="title">
       <string>Synchronization</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="noSyncRadioButton">
         <property name="text">
          <string>No sync</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="sonyRadioButton">
         <property name="text">
          <string>Sony 9 pin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="sonyFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="sonyCommunicationTimeCodeTypeLabel">
            <property name="text">
             <string>Communication framerate:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="sonyCommunicationTimeCodeTypeComboBox">
            <item>
             <property name="text">
              <string>23.98 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>24 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>25 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>29.97 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>30 fps</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="sonyVideoSyncTimeCodeTypeLabel">
            <property name="text">
             <string>Video sync framerate:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="sonyVideoSyncTimeCodeTypeComboBox">
            <item>
             <property name="text">
              <string>23.98 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>24 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>25 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>29.97 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>30 fps</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="ltcRadioButton">
         <property name="text">
          <string>LTC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="ltcFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="3" column="0">
           <widget class="QLabel" name="ltcInputPortLabel">
            <property name="text">
             <string>Audio input port:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="ltcInputPortComboBox"/>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="ltcTimeCodeTypeComboBox">
            <item>
             <property name="text">
              <string>23.98 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>24 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>25 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>29.97 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>30 fps</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="ltcTimeCodeTypeLabel">
            <property name="text">
             <string>Timecode type:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mtcRadioButton">
         <property name="text">
          <string>Midi timecode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="mtcFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="1" column="1">
           <widget class="QComboBox" name="mtcExistingInputPortComboBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="mtcExistingInputPortRadioButton">
            <property name="text">
             <string>Read from existing port:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="mtcVirtualInputPortRadioButton">
            <property name="text">
             <string>Read from virtual port:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="mtcVirtualInputPortLineEdit"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="mtcForce24as2398CheckBox">
            <property name="text">
             <string>Force 24 fps as 23.98 when reading MTC</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mmcCheckBox">
         <property name="text">
          <string>Midi machine control:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="mmcFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout_5">
          <item row="0" column="1">
           <widget class="QComboBox" name="mmcOutputPortComboBox"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="mmcOutputPortLabel">
            <property name="text">
             <string>Send MMC message to port:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="PhDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PhDialogButtonBox</class>
   <extends>QDialogButtonBox</extends>
   <header>PhCommonUI/PhDialogButtonBox.h</header>
  </customwidget>
  <customwidget>
   <class>PhLockableSpinBox</class>
   <extends>QSpinBox</extends>
   <header>PhCommonUI/PhLockableSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
